src/pages/landing/landingPage.js

```diff
@@ -17,22 +17,10 @@ class LandingPage extends React.Component {
   render() {
     const { walkthroughServices, middlewareServices, user } = this.props;
 
-    const LandingPageMastHead = () => (
```

Comment:
Do you think we should use this opportunity to remove landingPageMastHead.js as well as part of this change? I think it was obsolete even before this change.

Reply:
Good call - I've removed that file.